### PR TITLE
Address gcc9 warnings for a clean compile.

### PR DESCRIPTION
**Pull request by postwait**

### CODE REVIEW

File: src/utils/mtev_log.c [hunk 1]
```diff
@@ -1047,8 +1047,8 @@ jlog_logio_cleanse(mtev_log_stream_t ls) {
     if(is_datafile(entry->d_name, &logid) &&  /* make sure it is a datafile */
        logid < earliest &&                    /* and that is older enough */
        logid != log->current_log) {           /* and that isn't current */
-      char fullfile[PATH_MAX];
-      char fullidx[PATH_MAX];
+      char fullfile[PATH_MAX+PATH_MAX];
+      char fullidx[PATH_MAX+PATH_MAX];
```
Comment (pamaddox) [inline src/utils/mtev_log.c]: Why "PATH_MAX+PATH_MAX"?

Comment (postwait) [reply src/utils/mtev_log.c]: making sure there's enough room to snprintf later.

File: src/noitedit/search.c [hunk 2]
```diff
@@ -454,7 +454,7 @@ protected el_action_t
 cv_search(EditLine *el, int dir)
 {
 	char ch;
-	char tmpbuf[EL_BUFSIZ];
+	char tmpbuf[EL_BUFSIZ-3];
```
Comment (pamaddox) [inline src/noitedit/search.c]: Any particular reason for dropping the size here?

Comment (postwait) [reply src/noitedit/search.c]: to make sure it isn't too large to copy into the buffer it targets.

**Pull request merged**